### PR TITLE
EZP-28871: upload_max_filesize setting should be taken into account in the validation of uploaded files

### PR DESCRIPTION
**Pull request by adamwojs**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-28871
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

The second part of https://github.com/ezsystems/repository-forms/pull/218

#### Checklist:
- [ ] Wait until related PR will be merged
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/fieldtypes/edit/ezmedia.html.twig [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 {%- block ezplatform_fieldtype_ezmedia_row -%}
     {% set preview_block_name = 'ezmedia_preview' %}
-    {% set max_file_size = (form.parent.vars.value.fieldDefinition.validatorConfiguration.FileSizeValidator.maxFileSize * 1024 * 1024)|round %}
+    {% set max_file_size = min(form.parent.vars.value.fieldDefinition.validatorConfiguration.FileSizeValidator.maxFileSize * 1024 * 1024, max_upload_size)|round %}
```
Comment (kmadejski) [inline src/bundle/Resources/views/fieldtypes/edit/ezmedia.html.twig]: Missing `(` and `)`, should be as here: https://github.com/ezsystems/ezplatform-admin-ui/pull/368/files#diff-1a697e5a15cc6441ab21b051e917770dR7

File: src/bundle/Resources/views/fieldtypes/edit/ezbinaryfile.html.twig [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 {%- block ezplatform_fieldtype_ezbinaryfile_row -%}
     {% set preview_block_name = 'ezbinaryfile_preview' %}
-    {% set max_file_size = (form.parent.vars.value.fieldDefinition.validatorConfiguration.FileSizeValidator.maxFileSize * 1024 * 1024)|round %}
+    {% set max_file_size = min((form.parent.vars.value.fieldDefinition.validatorConfiguration.FileSizeValidator.maxFileSize * 1024 * 1024), max_upload_size)|round %}
```
Comment (kmadejski) [inline src/bundle/Resources/views/fieldtypes/edit/ezbinaryfile.html.twig]: Inconsistency: https://github.com/ezsystems/ezplatform-admin-ui/pull/368/files#diff-88aa7dd94356657989115af89ba83a35R7, maybe just remove unnecessary parentheses 😉

Comment (adamwojs) [reply src/bundle/Resources/views/fieldtypes/edit/ezbinaryfile.html.twig]: done 👍

**Comment by lserwatka** [comment]
@adamwojs could you change it to maser, we won't release 2.0.3.

**Comment by adamwojs** [comment]
@lserwatka Done.

**Pull request merged**